### DISCUSSION TITLE
HMS-800-2T openDTU blinkyparts with CMT3200  no connection -->solved

### DISCUSSION THREAD

**Discussion by zigit780** [tbnobody/OpenDTU#1470] [Q&A]
Hello I have in operation a openDTU with CMT3200 modul from blinkyparts. 
OpenDTU is working fine but I don't get a connection to the inverter.
In the Info of openDTU I see 
![image](https://github.com/tbnobody/OpenDTU/assets/69857197/38c14463-1b32-4f56-aef6-224b21ff3a7c)

Means this info  "CMT2300A Chip-Status | not connected  "  under info/system because no connection to the inverter or means it there is no internal communication to the radio-module possible?

I try to evaluate is there a problem with the radio-module of the openDTU or is it only connection topic to the inverter.

The inverter currently is working fine with the Hoymile DTU-Wlite S and sends data.

I have also switch off the original DTU WLite S but wasn't sucessful till now.

Also deleted and add the inverter again in the list. 

Newest SW on the openDTU. 
v23.10.9

**Comment by stefan123t** [comment]
Can you check whether GPIO3 from the CMT2300A / Blinky Parts board is connected to your ESP32 and double check its connection / wiring both physically using a Diode Tester *and* logically in the currently used pin_mappings.json ?

**Comment by zigit780** [comment]
Hi, so thanks for the hint to check again the electrical connections again.
The problem was that I haven't read carefully or maybe overlooked the picture in the manual from blinkyparts. 
The connector on the rf-module was soldered of the front-side but must be on the backside so all signals were mirrored on the board. 
But I had luck noting was damaged as the VCC was only mirrored to a digital signal-port.
Maybe a sentence in red letters to check that again would be good, as improvement in the manual.

So to answer also the question for others:

_**CMT2300A Chip-Status not conneted  --> means really that there is no connection to the rf-module or CMT3200A IC.**_

This could be also a good hint in the Troubleshoot-Guide.

So this problem is solved.

**Reply by se-wen** [reply]
Hi zigit780, I have a similar issue but I could not find any deviation according to the Manual Form blikyparts. Could you maybe create some photos from your (working) device so that I can compare ist with my one?

**Reply by zigit780** [reply]
No but look into the photo by step 13 if you have the pin soldered on the backside. That was my problem.

**Comment by zigit780** [comment]
![image](https://github.com/tbnobody/OpenDTU/assets/69857197/f2d351e9-6b54-46a1-a7cb-45ca6a28a481)

**Reply by se-wen** [reply]
Thanks, but thats correct on my site

**Comment by vcarre83** [comment]
I have the same problem with my opendtu blink CM2300A and ESP32
![opendtu cmt2300A](https://github.com/tbnobody/OpenDTU/assets/117444530/4cecf02f-18ba-48af-b27c-63f2c5a61772)

**Reply by vcarre83** [reply]
Hi
For the test I have buy a second opendtu with pre solder CMT3200 module and i have the same problem : not connected. I don't understand because i have two opendtu blinkparts with a error not connected.

**Comment by se-wen** [comment]
Hi, my issue was indeed also "CMT2300A Chip-Status not conneted --> means really that there is no connection to the rf-module or CMT3200A IC."

I really had little idea how to solder properly. After watching a few videos about soldering circuit boards and re-soldering all the solder joints, everything worked straight away.

**Reply by se-wen** [reply]
E.g. https://www.conrad.de/de/ratgeber/handwerk/richtig-loeten-lernen.html
